tests/TestCase.php

```diff
@@ -18,6 +19,8 @@ protected function setUp(): void
         Factory::guessFactoryNamesUsing(
             fn (string $modelName) => 'Cachet\\Database\\Factories\\'.class_basename($modelName).'Factory'
         );
+
+        $this->withoutApiRateLimiting();
```

Comment:
This has been added because running `composer test` ends up hitting the rate limit for the `api` guard set by workbench 😢

Reply:
@joelbutcher I ran into this myself and I wonder whether we should create a custom Rate Limit for Cachet?

Reply:
Can do? Could have it configurable to developers via the cachet config file, too

Reply:
We could set a custom middleware in the `api_middleware` config.

Reply:
@jbrooksuk Would be better to separate that from this PR as it's not a blocker for what we're adding here, but would be good to include regardless